schemas/PollEnd.json

```diff
@@ -0,0 +1,15 @@
+{
+  "type": "object",
+  "description": "Triggered when a poll has been ended.",
+  "properties": {
+    "eventName": {
+      "enum": ["PollEnd"]
+    },
+    "doNotUse": {
```

Comment:
Should this be removed before we merge?

Reply:
See: https://github.com/matrix-org/matrix-analytics-events/pull/85#issuecomment-1698162108
We can merge it and then remove this once the script that generates the Kotlin code is fixed.